pkg/buses/monitor.go

```diff
@@ -164,20 +164,10 @@ func (h MonitorEventHandlerFuncs) onProvision(channel *channelsv1alpha1.Channel,
 func (h MonitorEventHandlerFuncs) onUnprovision(channel *channelsv1alpha1.Channel, monitor *Monitor) error {
 	if h.UnprovisionFunc != nil {
 		err := h.UnprovisionFunc(channel)
-		channelCopy := channel.DeepCopy()
```

Comment:
It may be worth leaving a comment here noting that status update is intentionally omitted since the resource is deleted.